addons/mail/models/mail_render_mixin.py

```diff
@@ -312,7 +313,7 @@ def _render_template_qweb(self, template_src, model, res_ids,
                     group = self.env.ref('mail.group_mail_template_editor')
                     raise AccessError(_('Only users belonging to the "%s" group can modify dynamic templates.', group.name)) from e
                 _logger.info("Failed to render template : %s", template_src, exc_info=True)
-                raise UserError(_("Failed to render QWeb template : %s)", template_src)) from e
+                raise UserError(_("Failed to render QWeb template : %s)", traceback.format_exc())) from e
```

Comment:
Ok but now we don't know which template failed anymore. I would follow the logger and just append that at the end, assuming we do want this.